parser/dataTableSpecs_test.go

```diff
@@ -102,6 +103,50 @@ func TestGetSpecsForDataTableRows(t *testing.T) {
 	}
 }
 
+func TestGetSpecsForDataTableRowsShouldHaveEqualNumberOfScenearioInSpecsScenariosAndItemCollection(t *testing.T) {
```

Comment:
`spec.Items` consists of other items, ex, `comment`, `context`, `teardown`. So this expectation is incorrect.

I'd say here the test should ensure all scenarios are present in `Items`